packages/web/src/common/store/backend/sagas.ts

```diff
@@ -44,28 +42,7 @@ export function* waitForBackendSetup() {
   }
 }
 
-function* awaitReachability() {
-  const isNativeMobile = yield* getContext('isNativeMobile')
-  const isReachable = yield* select(getIsReachable)
-  if (isReachable || !isNativeMobile) return true
-  const { action } = yield* race({
-    action: take(reachabilityActions.SET_REACHABLE),
-    delay: delay(REACHABILITY_TIMEOUT_MS)
-  })
-  return !!action
-}
-
 export function* setupBackend() {
-  const establishedReachability = yield* call(awaitReachability)
-  // If we couldn't connect, show the error page
-  // and just sit here waiting for reachability.
-  if (!establishedReachability) {
-    console.error('No internet connectivity')
-    yield* put(accountActions.fetchAccountNoInternet())
```

Comment:
So we aren't dispatching this action anymore right? Not sure if it's necessary. But might be good to track down